src/components/inherited-map/components/filter-panel/date-filter.js

```diff
@@ -87,7 +87,12 @@ const DateFilterSection = ({ filter }) => {
           />
         </div>
         <div className="date-dropdown">
-          <button className="date-button" onClick={(e) => setShow(true)}>
+          <button
+            className="date-button"
+            onClick={() => {
+              setShow(true);
+            }}
```

Comment:
Replacing `() => x()` with `(x) => { x() }` in touched caes to avoid unexpected returns. I guess this will be caught by another rule we’ll re-enable later, so chaning the code now to reduce future diffs.